packages/now-static-build/src/frameworks.ts

```diff
@@ -121,6 +121,14 @@ export const frameworks: Framework[] = [
       },
     ],
   },
+  {
+    name: 'Scully',
+    slug: 'scully',
+    dependency: '@scullyio/init',
+    minNodeRange: '10.x',
+    buildCommand: 'ng build && scully',
+    getOutputDirName: async () => 'dist/static',
```

Comment:
Are you sure that the output is correct?

It seems like you should read angular.json as described here: https://mobile.twitter.com/styfle/status/1208073521190051844

Reply:
Right, this is just the default for Scully. I think the problem is that there can be multiple projects in one angular.json file with multiple output paths. I'll need to check further

Reply:
https://zeit.atlassian.net/browse/PRODUCT-935?atlOrigin=eyJpIjoiNGM2NzNhMDgwMmZjNGRhMGFkZjMzNmI1Nzg2ZDMwYTAiLCJwIjoiaiJ9